lib/ITG3205/ITG3205.h

```diff
@@ -1,57 +1,62 @@
 #include "Arduino.h"
 
 #ifndef __ITG3205_H_
-#define __ITG3205_H_
+# define __ITG3205_H_
 
 
-class ITG3205 {
-
-#define ITG3200_Address 0x68
-#define PWR_MGM 0x3E
-#define SMPLRT_DIV 0x15
-#define DLPF_FS 0x16
-#define INT_CFG 0x17
+# define ITG3200_Address  0x68
+# define ITG3200_WHOAMI   0x00
+# define PWR_MGM          0x3E
+# define SMPLRT_DIV       0x15
+# define DLPF_FS          0x16
+# define INT_CFG          0x17
 
-#define LSB_DEG 14.375
+# define LSB_DEG          14.375
 
+# define TEMP_OUT_H       0x1B
+# define TEMP_OUT_L       0x1C
 
-#define TEMP_OUT_H 0x1B
-#define TEMP_OUT_L 0x1C
+# define GYRO_XOUT_H      0x1D
+# define GYRO_XOUT_L      0x1E
 
-#define GYRO_XOUT_H 0x1D
-#define GYRO_XOUT_L 0x1E
+# define GYRO_YOUT_H      0x1F
+# define GYRO_YOUT_L      0x20
 
-#define GYRO_YOUT_H 0x1F
-#define GYRO_YOUT_L 0x20
+# define GYRO_ZOUT_H      0x21
+# define GYRO_ZOUT_L      0x22
 
-#define GYRO_ZOUT_H 0x21
-#define GYRO_ZOUT_L 0x22
+class ITG3205 {
+  uint8_t address;
 
-    byte address;
 public:
-    ITG3205(): address(ITG3200_Address) {}
-    typedef struct vector
-    {
-        int x, y, z, t;
-    } vector;
 
-    vector g; // gyro angular velocity readings
+  ITG3205() : address(ITG3200_Address) {}
+
+  ITG3205(uint8_t _address) : address(_address) {}
 
-    int offset[3];
+  typedef struct vector
+  {
+    int x, y, z, t;
+  } vector;
 
-    void calibrate();
-    void initGyro();
-    void GyroRead();
+  vector g; // gyro angular velocity readings
```

Comment:
Hmm this is a bit tricky name for the struct.
It is defined in the scope of the class, so it probably doesn't really matter, as you really must jump through some hoops to run into compiler issues.
Also any project that has `using namespace std;` in a header file deserve to fail to compile, but still it feels funky to see a struct named the same as a std container.

Reply:
That's inherited from the original library, and also the method naming is a bit dodgy/inconsistent, I'll change the naming to be a bit more consistent.